packages/ui/src/components/Badge/Badge.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { Badge, largeStyleProps, smallStyleProps } from './Badge';
+
+describe('Badge', () => {
+  it('renders correctly in large size', () => {
+    const text = 'Badge Text';
+
+    render(<Badge size="large">{text}</Badge>);
+
+    const badge = screen.getByText(text);
+
+    expect(badge).toHaveStyle(largeStyleProps);
+    expect(badge).toBeVisible();
```

Comment:
When you want to check if an element correctly renders in the DOM it's recommended to use the `toBeInTheDocument` method